stdlib/http/src/main/ballerina/src/http/resiliency/failover_client_endpoint.bal

```diff
@@ -308,7 +308,7 @@ function performFailoverAction (string path, Request request, HttpOperation requ
     Response inResponse = new;
     HttpFuture inFuture = new;
     Request failoverRequest = request;
-    error?[] failoverActionErrData = [];
```

Comment:
@ThisaruGuruge can you verify these changes please? The previous behaviour (passing an `error?[]` where a `ClientError?[]` is expected in `populateFailoverErrorHttpStatusCodes`) was allowed due to a bug.

Reply:
This should be fine.